autoload/lsp/options.vim

```diff
@@ -135,7 +135,10 @@ export var lspOptions: dict<any> = {
   customCompletionKinds: false,
 
   # A dictionary with all completion kinds that you want to customize
-  completionKinds: {}
+  completionKinds: {},
+
+  # Filter duplicate completion items
+  filterCompletionDuplicates: false,
```

Comment:
Can you update the documentation with a description about this new option?

Reply:
I've added to the lsp-options